pkg/transformers/namespace.go

```diff
@@ -97,5 +97,46 @@ func (o *namespaceTransformer) Transform(m resmap.ResMap) error {
 		}
 
 	}
+	o.updateClusterRoleBinding(m)
 	return nil
 }
+
+func (o *namespaceTransformer) updateClusterRoleBinding(m resmap.ResMap) {
+	samap := resmap.ResMap{}
+	for id := range m {
+		if id.Gvk().Kind == "ServiceAccount" {
```

Comment:
I suggest also checking `Group` and `Version` here, since `ServiceAccount` is a very common name and other groups may also have this `Kind`.

Reply:
Added.